sample/ios/sample.xcodeproj/project.pbxproj

```diff
@@ -868,6 +868,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
`l`: Why do we need this?

Reply:
I guess we have something similar on Flutter https://github.com/getsentry/sentry-dart/issues/348